components/site/Sidebar/index.js

```diff
@@ -5,21 +5,13 @@ import { Spinner } from '@twilio-paste/core/spinner';
 import { SidebarContainer } from './SidebarContainer';
 import { SideNav } from './SideNav';
 import { SideNavList } from './SideNavList';
-import { NavContext } from '../NavContext';
+import pages from '../../../pages/pages.json';
 
 const SideBar = () => {
-  const { nav, isLoading } = React.useContext(NavContext);
-
   return (
     <SidebarContainer>
       <SideNav>
-        {isLoading ? (
-          <Box display="flex" justifyContent="center">
-            <Spinner title="loading..." />
-          </Box>
-        ) : (
-          <SideNavList nav={nav} />
-        )}
+        <SideNavList nav={pages} />
```

Comment:
What will be rendered on the nav if `pages` is empty? Will it throw an error?

Reply:
This is a great call out ^

Potentially `SideNavList` doesn't handle this, and is a great candidate to write a test for 🎉

Reply:
@nkrantz when pages.json is an empty json ({}), the page renders without any navigation links. However, if the pages.json file is completely empty (nothing in it at all, not even curly braces), that throws an error. 

Potential solution for this in the next comment!

Reply:
> However, if the pages.json file is completely empty (nothing in it at all, not even curly braces), that throws an error

This I would try and accommodate for in throwing an error, or catching it to prevent someone doing it accidentally and not knowing what they did.

Giving them a nice friendly error message would be a great idea here.